App.tsx

```diff
@@ -32,169 +33,118 @@ Sentry.init({
 
 const queryClient: QueryClient = new QueryClient();
 
-const AvalancheCenterSelectorScreen = () => {
-  return (
-    <SafeAreaView style={styles.container}>
-      <AvalancheCenterSelector date={formatISO(new Date('2022-03-01'))} />
-    </SafeAreaView>
-  );
-};
+const Tab = createBottomTabNavigator<TabNavigatorParamList>();
+const HomeStack = createNativeStackNavigator<HomeStackParamList>();
 
-const SelectorStack = createNativeStackNavigator<StackParamList>();
-const AvalancheCenterSelectionScreen = () => {
-  // TODO(skuznets) not showing the header here means iOS has no way to get back to this screen once they choose a center, but showing it means we double-up on headers ... ?
-  return (
-    <SelectorStack.Navigator initialRouteName="avalancheCenterSelection" screenOptions={{headerShown: false}}>
-      <SelectorStack.Screen name="avalancheCenterSelection" component={AvalancheCenterSelectorScreen} options={{title: 'Select an Avalanche Center'}} />
-      <SelectorStack.Screen name="avalancheCenterHome" component={AvalancheCenterTabScreen} options={({route}) => ({title: route.params.center_id})} />
-    </SelectorStack.Navigator>
-  );
-};
+// TODO(brian): do we need this? I'm guessing this kept things working while
+// developing in the offseason. Also TBD whether this should be part of the
+// route; are there use cases for supporting forecasts from different dates than
+// "today"?
+const defaultDate = formatISO(new Date('2022-03-01'));
```

Comment:
@stevekuznetsov LMK if you have thoughts here!

Reply:
Yeah - I was hoping that we would allow a date picker in some sort of view - need design input but a calendar should suffice for now. Default to "today".